docs/content/docs/libs/state_processor_api.md

```diff
@@ -514,3 +512,120 @@ savepointWriter
         OperatorIdentifier.forUid("new-uid"))
     ...
 ```
+
+## Table API
+
+### Getting started
+
+Before getting started with state SQL connector, make sure to review our [Flink SQL](../development/sql.md) guidelines.
```

Comment:
we refer to "state SQL connector" before mentioning it . Maybe we could make this more general. Before you interrogate state using the table API, make sure to ....